frameworks/Elixir/phoenix/web/controllers/page_controller.ex

```diff
@@ -5,17 +5,20 @@ defmodule Hello.PageController do
 
   def index(conn, _params) do
     conn
+    |> put_resp_content_type("application/json", nil)
```

Comment:
If we are going to explicitly set the content type, then I would use send_resp and give up on the `json` and `text` functions altogether. What do you think?